tftypes/value_json_test.go

```diff
@@ -380,3 +380,47 @@ func TestValueFromJSON(t *testing.T) {
 		})
 	}
 }
+
+func TestValueFromJSONWithOpts(t *testing.T) {
+	t.Parallel()
+	type testCase struct {
+		value Value
+		typ   Type
+		json  string
+	}
+	tests := map[string]testCase{
+		"object-with-missing-attribute": {
```

Comment:
Nit: Can you include a "happy" path case as well, where it doesn't ignore any attributes?

Reply:
Included "happy" path.